src/Commands/core/UpdateDBCommands.php

```diff
@@ -112,6 +112,8 @@ public function entityUpdates($options = ['cache-clear' => true])
      *   description: Description
      *   type: Type
      * @default-fields module,update_id,type,description
+     * @filter-output
+     * @filter-default-field module
```

Comment:
How about a rule that defining @filter-default-field implies @filter-output? No need to do both, typically.

Reply:
Should be possible.

Reply:
Done